app/assets/stylesheets/_main.scss

```diff
@@ -562,5 +563,27 @@ body {
 .neighbour-letter-export {
   width: calc(430px - 1.5cm);
   letter-spacing: 0.01px;
+}
+
+.govuk-heading-xs {
```

Comment:
I'm not really sure where to put these. They're not _really_ part of the TOC, so I'm not sure they belong in its stylesheet; but they are also used (only) by the TOC.